sdcm/nemesis.py

```diff
@@ -4122,3 +4138,21 @@ class StartStopValidationCompaction(Nemesis):
 
     def disrupt(self):
         self.disrupt_start_stop_validation_compaction()
+
+
+class TopologyChangesMonkey(Nemesis):
+    """
+    Run only nemesis which change cluster topology:
+    - Decommission
+    - GrowShrinkCluster
+    - TerminateandReplaceNode
+    - TerminateRemoveNode
+    - DrainNode
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.disrupt_methods_list = self.get_list_of_methods_compatible_with_backend(topology_changes=True)
+
+    def disrupt(self):
+        self.call_random_disrupt_method(disrupt_methods=self.disrupt_methods_list)
```

Comment:
Why do we still need this and can't use "SisyphusMonkey" with nemesis_include_filter: ['topology_changes']?

Reply:
fixed